src/parser.ts

```diff
@@ -421,6 +421,10 @@ function variableDefinitions(): ast.VariableDefinitionNode[] | undefined {
     idx++;
     ignored();
     do {
+      let _description: ast.StringValueNode | undefined;
+      if (input.charCodeAt(idx) === 34 /*'"'*/) {
+        _description = value(true) as ast.StringValueNode;
```

Comment:
Would we want to add an invariant for checking that this can only be a string or split the string out?